scripts/correlation.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+
+def main():
+    df = pd.read_csv("./data/clean_data_without_outliers.csv")
```

Comment:
sjp zrj geras, bet radau prie ko prikibt vis tiek

```suggestion
    df = pd.read_csv("../data/clean_data_without_outliers.csv")
```